src/service/poc_lora.proto

```diff
@@ -27,6 +27,7 @@ enum invalid_reason {
   self_witness = 14;
   stale = 15;
   scaling_factor_not_found = 16;
+  generic_error = 17;
```

Comment:
Rename to "unknown". Generic makes it sound like it has a meaningful use and we're really trying to indicate it's an error of unknown origin.

Reply:
done, ocd satisfied